### PR TITLE
Lazy loading and new root_path default to '/'

### PR DESCRIPTION
**Pull request by osantana**
@erickwilder, please, take a look into this PR to fix Issue #8 .

### CODE REVIEW

File: docs/source/usage.rst [hunk 1]
```diff
@@ -152,10 +155,36 @@ of the ``starting_path`` as the ``root_path`` when instantiating ``Configuration
...
+
+    # Code example in project/app/settings.py
+    from prettyconf import config
+
+    config.starting_path = "/foo/bar"
+    config.root_path = "/baz"
+
+    MY_CONFIG = config("PROJECT_MY_CONFIG")  # raises an InvalidPath exception here
+
 The example above will start looking for files at ``project/app/`` and will stop looking
```
Comment (erickwilder) [inline docs/source/usage.rst]: This sentence makes no sense anymore, because the example above has changed.

Comment (erickwilder) [inline docs/source/usage.rst]: This paragraph is a little confusing. The example above has changed

**Comment by erickwilder** [comment]
Despite some docs with statements referencing the old examples. It's ok for me.

Not related question: why do we need to be at `0.5.0` if the status is stable? Can't we start to use a `1.0.0` version scheme?

**Comment by osantana** [comment]
I'll fix documentation and, after this, we can release a 1.0 version :)

**Pull request merged**